schemas/app-sre/app-1.yml

```diff
@@ -46,149 +56,190 @@ properties:
-
   architectureDocument:
     type: string
     format: uri
     description: |
-      The Architecture Document should be written with SREs as the target audience. Anyone
-      reading the Architecture Document should understand how the service works, even
-      without any prior background. This document will be the primary reference during
-      incidents. To ensure the Architecture Document is always accessible to SREs and its
-      changes properly reviewed and tracked, it must be written in markdown and stored in
-      a place where SREs can control its changes.
-
+      The URL to the architecture document for the application. This document 
+      should provide a detailed explanation of the application's architecture 
+      and be accessible to SREs.
   parentApp:
     "$ref": "/common-1.json#/definitions/crossref"
     "$schemaRef": "/app-sre/app-1.yml"
-
+    description: |
+      Reference to the parent application, if applicable.
   serviceOwners:
-    description: Teams or individuals who is/are responsible for the running instance of the software.
     type: array
+    description: |
+      A list of teams or individuals responsible for the running instance of 
+      the application.
     items:
       type: object
       additionalProperties: false
       properties:
         name:
           type: string
+          description: |
+            The name of the service owner.
         email:
           type: string
           format: email
+          description: |
+            The email address of the service owner.
       required:
       - name
       - email
-
   serviceNotifications:
-    description: Teams or individuals who is/are should get notifications related to the service.
     type: array
+    description: |
+      A list of teams or individuals who should receive notifications related 
+      to the application.
     items:
       type: object
       additionalProperties: false
       properties:
         name:
           type: string
+          description: |
+            The name of the notification recipient.
         email:
           type: string
           format: email
+          description: |
+            The email address of the notification recipient.
       required:
       - name
       - email
-
   serviceDocs:
-    description: List of service docs
     type: array
+    description: |
+      A list of links to service documentation for the application.
     items:
-      description: Service document link
       type: string
-
+      description: |
+        The URL of a service document.
   dependencies:
     type: array
+    description: |
+      A list of dependencies required by the application.
     items:
       "$ref": "/common-1.json#/definitions/crossref"
       "$schemaRef": "/dependencies/dependency-1.yml"
-
   gcrRepos:
     type: array
+    description: |
+      A list of Google Container Registry (GCR) repositories associated with 
+      the application.
```

Comment:
GCR is deprecated, now it's called Artifact Registry, https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr